sections/multirow.liquid

```diff
@@ -0,0 +1,437 @@
+{{ 'component-image-with-text.css' | asset_url | stylesheet_tag }}
+
+{%- style -%}
+  .section-{{ section.id }}-padding {
+    padding-top: {{ section.settings.padding_top | times: 0.75 | round: 0 }}px;
+    padding-bottom: {{ section.settings.padding_bottom | times: 0.75 | round: 0 }}px;
+  }
+
+  @media screen and (min-width: 750px) {
+    .section-{{ section.id }}-padding {
+      padding-top: {{ section.settings.padding_top }}px;
+      padding-bottom: {{ section.settings.padding_bottom }}px;
+    }
+  }
+{%- endstyle -%}
+
+{%- liquid
+  assign odd_class = 'image-with-text__grid--standard'
+  assign even_class = 'image-with-text__grid--reverse'
+  if section.settings.image_layout contains 'right'
+    assign odd_class = 'image-with-text__grid--reverse'
+    assign even_class = 'image-with-text__grid--standard'
+  endif
+
+  assign transparent_content = false
+  if section.settings.row_color_scheme == section.settings.section_color_scheme
+    assign transparent_content = true
+  endif
+
+  if transparent_content and settings.text_boxes_shadow_opacity == 0 and settings.text_boxes_border_thickness == 0 or settings.text_boxes_border_opacity == 0
+    assign padding_class = ' collapse-padding'
+  endif
+  if settings.text_boxes_border_thickness > 0 and settings.text_boxes_border_opacity > 0 and settings.media_border_thickness > 0 and settings.media_border_opacity > 0
+    assign borders_class = ' collapse-borders'
+  endif
+  unless transparent_content and settings.media_border_thickness > 0 and settings.text_boxes_shadow_opacity == 0 and settings.text_boxes_border_thickness == 0 or settings.text_boxes_border_opacity == 0
+    assign corners_class = ' collapse-corners'
+  endunless
```

Comment:
That's fair. Your PR, so ultimately your call 😄 

If there's anything else you think we can do to make the code more reasonable, even if it's like breaking up the code over multiple lines or something, consider it. Otherwise, I'm fine shipping the way it is.